docs/en/tutorials/0_config.md

```diff
@@ -94,7 +94,17 @@ configs/{topic}/{task}/{algorithm}/{dataset}/{backbone}_[model_setting]_{dataset
       step=[170, 200])  # Steps to decay the learning rate
   total_epochs = 210  # Total epochs to train the model
   log_config = dict(  # Config to register logger hook
-      interval=50,  # Interval to print the log
+      intervlog_config = dict(  # Config to register logger hook
+    interval=50,  # Interval to print the log
+    hooks=[
+      dict(type='TextLoggerHook', by_epoch=False),  # The logger used to record the training process
+      dict(type='TensorboardLoggerHook', by_epoch=False),  # The Tensorboard logger to record log offline
```

Comment:
Why `by_epoch` is set to False by default?

Reply:
I used this parameter to show the possibility of changing it to True or False to those who want to use it.  It is possible to give it the True default value, I don't have any special reason to use the False value as default.

Reply:
I think it's better not to change the default setting here. To show the options, we can add `by_epoch=True` here.